README.md

```diff
@@ -30,8 +30,10 @@ the source files.
 This site uses a shared cloud.gov style, [cg-style](https://github.com/18F/cg-style). This means any styling code has to be developed in *cg-style*.
 
 1. Download or clone the *cg-style* repository, `git clone git@github.com:18F/cg-style.git`
-2. Run the watching build task in the *cg-style* repository: `npm run watch`
-3. Run `npm install` in the *cg-landing* repository.
-4. Run `npm link` in *cg-landing*.
-5. Run the watching build task in *cg-landing* repository: `npm run watch`
-6. Edit code in the *cg-style* directory and they will propagate down to *cg-landing*
+2. Follow the instructions on the [cg-style readme](https://github.com/18F/cg-style#install-and-use) to install and build the library.
+3. Run the watching build task in the *cg-style* repository: `npm run watch`
+4. Replace the `gem 'cloudgov-style'` line in the `Gemfile` with `'cloudgov-style', :github => '18F/cg-style-gem', :branch => 'master'`.
+6. Run `bundle` again.
+7. Run `bundle config local.cloudgov-style ~/path/to/cg-style/repo` wih the path to where you cloned the cg-style repo.
```

Comment:
So in that case, we'd just tell them to add `, :path => './vendor/extracted_library'` rather then change the whole line? That makes more sense